frontend/src/lib/components/EditableText.svelte

```diff
@@ -0,0 +1,157 @@
+﻿<script lang="ts">
+	import { LoadingIcon } from '$lib/icons';
+	import { ZodString, z } from 'zod';
+	import { _changeProjectName } from '../../routes/project/[project_code]/+page';
+	import IconButton from './IconButton.svelte';
+	import { Form, lexSuperForm, lexSuperValidate } from '$lib/forms';
+
+	export let value: string | undefined | null = undefined;
+	export let disabled = false;
+	export let saveHandler: (newValue: string) => Promise<unknown>;
+	export let placeholder: string | undefined = undefined;
+	export let multiline = false;
+	export let validation: ZodString | undefined = undefined;
```

Comment:
I think this should be marked as `const` because right now when the component is created the value from this is used to create the `formSchema` but is not updated if `validation` changes from whoever is using the component, which is totally fine, I'm not sure why it would change, but we should mark it as const to prevent it from being changed.